src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -75,4 +81,13 @@ class CallActor(call: Call, locallyQualifiedInputs: Map[String, WdlValue], backe
       case Failure(e) => failedInstantiation(e)
     }
   }
+
+  def receiveResults(results: Try[Map[String, WdlValue]]): Unit = {
```

Comment:
It feels a bit weird that both branches of the above effectively coalesce to call this and then it splits out success and failure again, perhaps the logic here can be inlined with the above if the previous suggestion is taken.